src/Parsers/GetObject/Multiple/Streamed.php

```diff
@@ -0,0 +1,93 @@
+<?php namespace PHRETS\Parsers\GetObject\Multiple;
+
+use GuzzleHttp\Psr7\Response;
+use PHRETS\Http\Response as PHRETSResponse;
+use Illuminate\Support\Collection;
+use PHRETS\Parsers\GetObject\Single;
+
+class Streamed
+{
+    // in bytes
+    // TODO: parameterize this
+    private const CHUNK_SIZE = 20 * 1000 * 1000;
```

Comment:
_potential blocker_

PHRETS currently depends on `>=5.5.0`, which means this line would have to be

```suggestion
    const CHUNK_SIZE = 20000000;
```

but since this is our fork, I feel like we should probably just bump the min php version requirement to 7.1 and take advantage of all the nice things. Troy is never going to merge our changes in anyway.

Reply:
I'd say the benefits of having them (at least the things you've pointed out) are pretty minor in comparison to the drawback of making it harder to upstream our changes, but I could go either way

Reply:
i don't have a strong preference, tbh